docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -118,3 +118,73 @@ Then, replace the contents of `pages/index.js` with the code below, restart the
   </TabItem>
 </Tabs>
 
+### Connected Components
+
+<Tabs>
+  <TabItem title="Next.js 13.4+">
+
+To use Connected componnets in app directory, make sure you re-export `Authenticator` as a Client Component by placing "use client" at the top of the file and set up with `Amplify.configure`, for example:
```

Comment:
This reads like re-exporting the `Authenticator` is going to allow for usage of any Connected Component?

Reply:
You are right... I am kinda biased when using Authenticator as an example